docs/refs.bib

```diff
@@ -765,6 +765,17 @@ @misc{Molina_2012_Optimal
   primaryClass={quant-ph}
 }
 
+@article{muller_lennert_renyi_2013,
```

Comment:
Please refer to the other references in this file for how to format the bibtex key.